config/routes.d/developer.rackspace.com.json

```diff
@@ -41,6 +41,7 @@
             "^/docs/dedicated-load-balancers/v2/dev-guide/": "docs-singlepage.html",
             "^/docs/glossary": "docs-singlepage.html",
             "^/docs/private-cloud": "user-guide.html",
+            "^/docs/private-cloud/release-notes": "docs-singlepage.html",
```

Comment:
This path doesn't match the path that you've mapped in content.d above (`/docs/private-cloud/rpc/v13/release-notes`). I'm guessing that this is unintentional?